### ISSUE TITLE
"Native Not Found" error message installing DW Agent

### ISSUE THREAD

**Issue by dan-willett** [dwservice/agent#26]
I just purchased a new Windows 10 laptop, and I am trying to install DW Agent on it, and it fails with the error message "Native Not Found"

This is a particularly unhelpful error message.   What is the problem and how do I fix it.

You may want to consider re-writing that error message so it is more helpful...

**Comment by dan-willett** [comment]
I was successfully able to install after getting a windows update.   However, could somebody shed some light on this error message?

**Comment by trpapp** [comment]
Yes, I've gotten that error as well and I deduced it to a corrupt download or not a fully updated windows environment. I think this has to do with how python (or the code in general) accesses the windows api.
